src/css/About.css

```diff
@@ -0,0 +1,77 @@
+.Title{
+    margin-top: 60%;
+    width: 100%;
+    text-align: right;
+    font-family: Medium Whyte Inktrap;
+    font-size: 127px;
```

Comment:
Change this to 13.5vh. We are switching to using vh now. vh is like px but it dynamically resizes the text based on the screen's height. So the size will increase as the screen size increases and decrease as the screen decreases.